src/Diagnostics.ModelsAndUtils/Models/CompilerResponse.cs

```diff
@@ -8,10 +8,14 @@ public class CompilerResponse
     {
         public bool CompilationSucceeded;
 
-        public IEnumerable<string> CompilationOutput;
+        public IEnumerable<string> CompilationTraces;
 
         public string AssemblyBytes;
 
         public string PdbBytes;
+
+        public string AssemblyName;
+       
+        public bool IsCompiled;
```

Comment:
Should we call this CompilationStateChanged ?
Alternatively, do we need this member. We can also deduce this using script etag from response header

Reply:
As discussed, I will remove the IsCompiled property and the server will always return script etag.. The UI will check if it has changed by comparing the previous etag and the current etag and then update the assembly name.